### ISSUE TITLE
Babel 7

### ISSUE THREAD

**Issue by fluky** [shuhei/babel-plugin-angular2-annotations#31]
Any plans to update this for Babel 7?

**Comment by georgediaz88** [comment]
Yes, any updates on this or alternative way to get this plugin compatible with Babel7?

**Comment by shuhei** [comment]
I don't have a plan to update this repo for now. Pull requests are always welcome 😃
